### PR TITLE
Styling project, just a basic nav bar at the left

### PR DESCRIPTION
**Pull request by vado21**

### CODE REVIEW

**Review by kassidywebb — CHANGES_REQUESTED** [review]
It looks like there's a lot of formatting errors that need to be fixed.

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -14,19 +14,46 @@
...
 /* Keep navigation menu on one line. */
+
 nav li {
-  display: inline-block;
-  margin-right: 10px
+    display: inline-block;
+    margin-right: 10px
 }
+
+nav li::hover {
```
Comment (kassidywebb) [inline src/main/webapp/css/main.css]: here

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -14,19 +14,46 @@
...
+    background: white;
+}
+
+nav li a {
+    color: white;
+    text-decoration: none;
+}
```
Comment (kassidywebb) [inline src/main/webapp/css/main.css:59]: here

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -16,24 +16,24 @@
...
-    You could start by modifying this page to tell the world more about your team.</p>
+    <p>CodeU Starter Project</p>
+    <p> Hello World!! This is the CodeU starter project. Click the links above to login and visit your page. You can post messages on your page, and you can visit other user pages if you have their URL.</p>
+    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative! You could start by modifying this page to tell the world more about your team.</p>
     <p>Im Editing</p>
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (kassidywebb) [inline src/main/webapp/index.html:39]: here (this means add a new line)

Comment (vado21) [reply src/main/webapp/index.html:39]: I dont know it does that automatically when I save the file in visual studio, I can not make the change.

**Pull request merged**